### ISSUE TITLE
No such file: "...\markdownlint-config-schema.json"

### ISSUE THREAD

**Issue by KillyMXI** [DavidAnson/vscode-markdownlint#169]
I get this warning when opening `.markdownlint.json` or `.markdownlint-cli2.jsonc`:

```
Problems loading reference 'file:///c%3A/Users/.../.vscode/extensions/davidanson.vscode-markdownlint-0.42.0/node_modules/markdownlint/schema/markdownlint-config-schema.json': 
Unable to load schema from 'c:\Users\...\.vscode\extensions\davidanson.vscode-markdownlint-0.42.0\node_modules\markdownlint\schema\markdownlint-config-schema.json': 
ENOENT: no such file or directory, open 'c:\Users\...\.vscode\extensions\davidanson.vscode-markdownlint-0.42.0\node_modules\markdownlint\schema\markdownlint-config-schema.json'.(768)
```

and no autocompletion inside the `config` section.

**Comment by DavidAnson** [comment]
Weird, I'll try this tonight. If you are bored, I'd be interested if this still happens for you when using earlier extension versions - especially 0.39.*.

**Comment by KillyMXI** [comment]
I've just checked it, and the issue only appears in 0.42.0.
Version 0.41.1 seems fine.

**Comment by DavidAnson** [comment]
My guess is the updated vsce dependency stopped including that file in the package. Will investigate tonight.

**Comment by DavidAnson** [comment]
Confirmed: `vsce` stopped including `node_modules/markdownlint/schema/markdownlint-config-schema.json` in the package between `1.91.0` and `1.92.0`. They don't seem to have a changelog, so it's hard to say why. I'll fix this shortly.

**Comment by craigmac** [comment]
Confirmed here, been driving me crazy the last two days and now realized the extension was updated in VS Code and so checked here. Here's the error I get:

```
[10:24:17 AM] ERROR: Exception while linting:
Error: ENOENT: no such file or directory, open '/Users/cmaceach/git/devx/docs/.markdownlint.jsonc
```

File does exist at that location. For what it is worth the file/s that I'm trying to lint when this happens are actually in a symlink folder, if that makes a difference?
